diffengine/__init__.py

```diff
@@ -431,14 +436,29 @@ def get_initial_config():
         consumer_secret = input("What is the consumer secret? ")
 
         token = request_pin_to_user_and_get_token(consumer_key, consumer_secret)
-
+        config["publishing_handlers"].append("Twitter")
         config["twitter"] = {
             "consumer_key": consumer_key,
             "consumer_secret": consumer_secret,
+            "token": token
```

Comment:
Hey, this shouldn't be here. Both `access_token` and the `access_token_secret` need to be inside the twitter account config at `feeds.{n].twitter` as you could have several accounts (as in my case at [diffbots](https://github.com/nahuelhds/diffbots)